tests/components/androidtv/test_media_player.py

```diff
@@ -127,7 +136,10 @@ def _setup(config):
         patch_key = "server"
 
     host = config[DOMAIN][CONF_HOST]
-    if config[DOMAIN].get(CONF_DEVICE_CLASS) != "firetv":
+    # CONF_NAME available for configuration imported from YAML
+    if conf_name := config[DOMAIN].get(CONF_NAME):
+        entity_id = slugify(conf_name)
```

Comment:
I'd recommend setting the entity_id as a test parameter instead similar to how we parametrize the config. We don't want conditional logic in tests.

Reply:
Ok, I will create a PR for this when I will have some more available time.